core/src/test/scala/com/pingcap/tispark/TiBatchWriteSuite.scala

```diff
@@ -33,15 +33,15 @@ class TiBatchWriteSuite extends BaseTiSparkTest {
       "SUPPLIER" ::
       Nil
 
-  private val batchWriteTablePrefix = "BATCH_WRITE"
+  private val batchWriteTablePrefix = "BATCH.WRITE"
 
   override def beforeAll(): Unit = {
     super.beforeAll()
     database = tpchDBName
     setCurrentDatabase(database)
     for (table <- tables) {
-      tidbStmt.execute(s"drop table if exists ${batchWriteTablePrefix}_$table")
-      tidbStmt.execute(s"create table if not exists ${batchWriteTablePrefix}_$table like $table ")
+      tidbStmt.execute(s"drop table if exists `${batchWriteTablePrefix}_$table`")
```

Comment:
how about we make `${batchWriteTablePrefix}_$table` constant?

Reply:
nice